shoov/modules/custom/shoov_ui_build/shoov_ui_build.module

```diff
@@ -64,3 +71,27 @@ function shoov_ui_build_create_pr($node) {
 
   return drupal_http_request($url);
 }
+
+/**
+ * Trigger "New build" event when bew ui_build is created.
+ *
+ * Implements hook_node_insert().
+ */
+function shoov_ui_build_node_insert($node) {
+  if ($node->type != 'ui_build') {
+    return;
+  }
+
+  $wrapper = entity_metadata_wrapper('node', $node);
+  $repo_id = $wrapper->og_repo->value(array('identifier' => TRUE));
+
+  // todo: Discover how to check actual user's access, not admin.
+  $account = user_load(1);
```

Comment:
@amitaibu Maybe there is some hook that happens after records are already in the node_access table?